### PR TITLE
Ctrl+Tab, Ctrl+Shift+Tab to change tabs

### PR DESCRIPTION
**Pull request by goutamnair7**

### CODE REVIEW

File: webactivity.py [hunk 1]
```diff
@@ -530,6 +530,22 @@ def _key_press_cb(self, widget, event):
...
                 self._tabbed_view.close_tab()
+            elif key_name == "Tab":
+                _logger.debug('keyboard: next tab')
+                current_index = self._tabbed_view.get_current_page()
+                if current_index == self._tabbed_view.get_n_pages() - 1:
+                    self._tabbed_view.set_current_page(0)
+                else:
+                    self._tabbed_view.set_current_page(current_index + 1)
+            elif event.get_state() & Gdk.ModifierType.SHIFT_MASK:
+                if key_name == "ISO_Left_Tab":
```
Comment (SAMdroid-apps) [inline webactivity.py]: Is there a reason to use `Tab` up the top and `ISO_Left_Tab` down here?

Comment (goutamnair7) [inline webactivity.py]: No, there is no particular reason. Even Pippy had the same convention so I decided to use it here. Is there a problem?

Comment (SAMdroid-apps) [inline webactivity.py]: No, just wondering :)

**Comment by SAMdroid-apps** [comment]
@godiard @tchx84 This is pretty merge-worthy stuff!

**Comment by SAMdroid-apps** [comment]
:+1:

**Comment by godiard** [comment]
Merged!

**Pull request merged**